bindings/matrix-sdk-crypto-nodejs/src/machine.rs

```diff
@@ -407,4 +435,16 @@ impl OlmMachine {
     pub async fn sign(&self, message: String) -> types::Signatures {
         self.inner.sign(message.as_str()).await.into()
     }
+
+    /// Shut down the `OlmMachine`.
+    ///
+    /// The `OlmMachine` cannot be used after this method has been called,
+    /// otherwise it will panic.
+    ///
+    /// All associated resources will be closed too, like the crypto storage
+    /// connections.
```

Comment:
We must add that the user is responsible to not use any objects that came from the `OlmMachine` after this `close` method has been called.

Reply:
Let's explain this and if you prefer this to be exposed I'm fine with it going in.